### ISSUE TITLE
iniflags: unknown flag names

### ISSUE THREAD

**Issue by sknebel** [jcjones/ct-mapreduce#3]
with the configuration as suggested `ct-fetch` doesn't start:
```
2017/08/16 19:55:33 iniflags: unknown flag name=[geoipDbPath] found at line [2] of file [./ct-fetch.conf]
2017/08/16 19:55:33 iniflags: unknown flag name=[runForever] found at line [10] of file [./ct-fetch.conf]
```
after commenting out the two lines it runs

**Comment by groovecoder** [comment]
I get the same error.

**Comment by jcjones** [comment]
The Go flags library I'm misusing for this kinda sucks; it's brittle as it assumes it should know every flag.

**Comment by groovecoder** [comment]
work-around: remove those line.

**Comment by jcjones** [comment]
Resolved by 9c0c036eae30dc1bea072276e4926983e0bba1d9
